inc/saitunnel.h

```diff
@@ -229,7 +229,8 @@ typedef enum _sai_tunnel_attr_t
     SAI_TUNNEL_ATTR_UNDERLAY_INTERFACE,
 
     /** tunnel overlay interafce [sai_object_id_t] (MANDATORY_ON_CREATE|CREATE_ONLY)
-     *  overlay interface is router interface when SAI_TUNNEL_ATTR_TYPE = SAI_TUNNEL_IPINIP or SAI_TUNNEL_IPINIP_GRE */
+     *  overlay interface is router interface when SAI_TUNNEL_ATTR_TYPE = SAI_TUNNEL_IPINIP or SAI_TUNNEL_IPINIP_GRE
+     *  overlay interface is bridge port object when SAI_TUNNEL_ATTR_TYPE = SAI_TUNNEL_VXLAN */
```

Comment:
meeting minutes: in dot.1d use case, the overlay interface is derived from vni to bridge port mapping. this is only used for dot1q, use case.